src/Chapter04.Tests/Listing04.25.RelyingOnIndentation.Tests.cs

```diff
@@ -8,8 +8,8 @@ public class ProgramTests
         [TestMethod]
         public void MainTest()
         {
-            const string expected = 
-@"The area of the circle is: 0.00";
+            string expected = 
+$@"The area of the circle is: {0.00M}";
```

Comment:
Does this need to be a decimal rather than a double?
```suggestion
$@"The area of the circle is: {0.00}";
```

Reply:
So I tried that for both 4.25 and 4.26 and the tests failed due to actual code producing 0.00 and the change created an expected value of 0.